library/Rules/NumericVal.php

```diff
@@ -13,8 +13,16 @@
 
 namespace Respect\Validation\Rules;
 
-class NumericVal extends AbstractRule
+/**
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
```

Comment:
Would be nice to have a single-line documentation here (which can be found in the documentation file of this rule).